Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Switchcounter Workshop
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '30')),
+    parameters([
+        stringParam(defaultValue: 'http://downloads.creatordev.io/creator-openwrt/v1/pistachio/marduk/creator-sdk-1.tar.bz2',
```

Comment:
I believe the SDK v2 did have a major change from v1, https://github.com/CreatorDev/openwrt/commit/e973475ee378c9cf57ded93600ff5c7eb64a9565, so highly recommend to use v2